presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -461,6 +465,23 @@ class SystemConfig : public ConfigBase {
   int32_t oldTaskCleanUpMs() const;
 };
 
+/// Provides access to connector properties defined in
+/// catalog/$connector_name.properties file.
+class ConnectorConfig : public ConfigBase {
+ public:
+  static constexpr std::string_view kConnectorName{"connector.name"};
+  static constexpr std::string_view kHiveImmutablePartitions{
```

Comment:
Btw, let's maybe add
  static constexpr const char* kOrcUseColumnNames = "hive.orc.use-column-names";
too?

Reply:
I plan to add the rest of the config all together later because we're missing a lot, do you think this one is especially important?

Reply:
Not really, just the one I know of. :)